docs/visualization/plotting-overview.ipynb

```diff
@@ -0,0 +1,256 @@
+    "d.coords[Dim.Y] = sc.Variable([Dim.Y], values=np.arange(N), unit=sc.units.us)\n",
+    "\n",
+    "d['data'] = sc.Variable([Dim.Y],\n",
+    "                        values=100*np.random.rand(N)+50,\n",
+    "                        unit=sc.units.counts)\n",
+    "d['data_with_errors'] = sc.Variable([Dim.Y],\n",
+    "                                    values=50*np.random.rand(N),\n",
+    "                                    variances=50*np.random.rand(N),\n",
+    "                                    unit=sc.units.counts)\n",
+    "\n",
+    "d['data_2d'] = sc.Variable([Dim.X, Dim.Y],\n",
+    "                           values=10.0*np.random.rand(M, N))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Plotting functionality is available in the `scipp.plot` module, with the `plot` function.\n",
+    "The information in a data array or dataset is typically enough to create meaningful plots:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from scipp.plot import plot\n",
+    "\n",
+    "plot(d)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Plotting slices or items of a dataset\n",
+    "\n",
+    "The usual indexing and slicing can be used to create plots of slices of data, or plots of individual items from a dataset.\n",
+    "\n",
+    "### Plot single entry of a dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot(d['data'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plot slice range"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot(d[Dim.Y, 4:7])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plot 1-D slice of 2-D data\n",
+    "\n",
+    "When slicing without a range the dimensionality reduces.\n",
+    "This can be used to, e.g., plot a 1-D slice through 2-D data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot(d[Dim.X, 4])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Logarithmic scale\n",
+    "\n",
+    "1-D data can be plotted on a logarithmic scale on one or both axes:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "d_1d = d.copy()\n",
+    "del d_1d['data_2d']\n",
+    "\n",
+    "plot(d_1d, logx=True)\n",
+    "plot(d_1d, logy=True)\n",
+    "plot(d_1d, logxy=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Axis labels and axis order\n",
+    "\n",
+    "By default scipp uses coordinate values to label the axes.\n",
```

Comment:
```suggestion
    "By default `scipp` uses coordinate values to label the axes.\n",
```

Reply:
Intentional, in line with how it is used in the rest of the doc pages.